scripts/pi-hole/js/messages.js

```diff
@@ -57,6 +57,22 @@ function renderMessage(data, type, row) {
         "</pre> to get the group configuration for this client."
       );
 
+    case "HOSTNAME":
+      var hint = new Array(row.blob2 + row.message.length + 3).join(" ");
+      return (
+        "Host names contains invalid character <code> " +
```

Comment:
Typo `contains` :)

Reply:
Also should probably be `hostnames`?

Reply:
If that's all you can come up with, then today I have successfully rebased something ;) I was just about to ping you and get you to doublecheck before  I did anything else.

Though I need to make sure I've not lost some other stuff first...

Reply:
Wait. There is no typo in `contains`...

Reply:
Agree with the latter though

Reply:
It's plural, so it should be contain?